src/pudl/output/ferc1.py

```diff
@@ -2774,3 +2792,70 @@ def nodes_to_df(calc_forest: nx.DiGraph, nodes: list[NodeId]) -> pd.DataFrame:
     except AttributeError:
         tags = pd.DataFrame()
     return pd.concat([index, tags], axis="columns")
+
+
+def out_ferc1__yearly_rate_base(
+    exploded_balance_sheet_assets_ferc1: pd.DataFrame,
+    exploded_balance_sheet_liabilities_ferc1: pd.DataFrame,
+    core_ferc1__yearly_operating_expenses_sched320: pd.DataFrame,
+) -> pd.DataFrame:
+    """Make a table of granular utility rate-base data.
+
+    This table contains granular data consisting of what utilities can
+    include in their rate bases. This information comes from two core
+    inputs: ``exploded_balance_sheet_assets_ferc1`` and
+    ``exploded_balance_sheet_liabilities_ferc1``. These tables include granular
+    data from the nested calculations that are build into the accounting tables.
+    See :class:`Exploder` for more details.
+
+    This rate base table also contains one specific addition from
+    :ref:`core_ferc1__yearly_operating_expenses_sched320`. In standard ratemaking
+    processes, utilities are enabled to include working capital - sometimes referred
+    to as cash on hand or cash reverves. A standard ratemaking process is to consider
+    the available rate-baseable working capital to be one eigth of the average
+    operations and maintenance expense. This function grabs that expense and
+    concatenates it with the rest of the assets and liabilities from the granular
+    exploded data.
+
+    """
+    # get the factoid name to grab the right part of the table
+    xbrl_factoid_name = pudl.transform.ferc1.FERC1_TFR_CLASSES[
+        "core_ferc1__yearly_operating_expenses_sched320"
+    ]().params.xbrl_factoid_name
+    # First grab the cash on hand out of the operating expense table.
+    # then prep it for concating. Calculate cash on hand & add tags
+    cash_working_capital = (
+        core_ferc1__yearly_operating_expenses_sched320[
+            core_ferc1__yearly_operating_expenses_sched320[xbrl_factoid_name]
+            == "operations_and_maintenance_expenses_electric"
+        ]
+        .assign(
+            dollar_value=lambda x: x.dollar_value / 8,
+            xbrl_factoid="cash_on_hand",  # newly definied (do we need to add it anywhere?)
+            tags_rate_base_category="net_working_capital",
+            tags_aggregatable_utility_type="electric",
+            table_name="core_ferc1__yearly_operating_expenses_sched320",
+        )
+        .drop(columns=[xbrl_factoid_name])
+        # the assets/liabilites both use ending_balance for its main $$ column
+        .rename(columns={"dollar_value": "ending_balance"})
+    )
```

Comment:
@jrea-rmi  this is the little process to grab the `operations_and_maintenance_expenses_electric` line from the operating expense table and convert it into working capital/cash on hand. Before adding it to the `in_rate_base` exploded data. Does this look right to you generally? Do the tags look okay & sufficient? I figured this should be `in_service` as plant status or have a plant function. but lmk it's obviously easy to add in there as an additional column.

Reply:
Yes this looks right to me generally!

I'd go with xbrl_factoid as "cash_working_capital" rather than "cash_on_hand".

I don't know if it needs plant status or plant function; my initial thought is those don't need to be added. I thought in_service etc. only applied to utility plant assets, and plant function only applied to plants in service. But I could see an expanded definition of those if you think there should be.

Reply:
okay great! I'll change it to "cash_working_capital". And I think your right on the status and function - or at least that sounds plausible to me. I just wanted to make sure!